pom.xml

```diff
@@ -119,6 +117,198 @@
+        <JavaAPIforKml.version>2.2.1_CODICE_1</JavaAPIforKml.version>
+        <javax.annotation-api.version>1.3</javax.annotation-api.version>
+        <javax.inject.version>1</javax.inject.version>
+        <javax.servlet-api.version>3.1.0</javax.servlet-api.version>
+        <javax.ws.rs.version>2.1</javax.ws.rs.version>
+        <javax-el.version>3.0.0</javax-el.version>
+        <javax-mail.version>1.5.6</javax-mail.version>
+        <jaxb.version>2.2.11</jaxb.version>
+        <jaxb2.version>0.9.4</jaxb2.version>
+        <jboss-logging.version>3.3.1.Final</jboss-logging.version>
+        <jcodec.version>0.2.0_1</jcodec.version>
+        <jdom.bundle.version>1.1_4</jdom.bundle.version>
+        <jetty.solr.version>9.4.10.v20180503</jetty.solr.version>
+        <jetty.version>9.4.11.v20180605</jetty.version>
+        <jgroups.version>3.6.13.Final</jgroups.version>
+        <joda-convert.version>1.7</joda-convert.version>
+        <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
+        <joda-time.version>2.10</joda-time.version>
+        <johnzon.version>0.9.5</johnzon.version>
+        <jpeg2000.version>1.3.1_CODICE_3</jpeg2000.version>
+        <jsonassert.version>1.5.0</jsonassert.version>
+        <json-simple.version>1.1.1</json-simple.version>
+        <json-smart.version>2.3</json-smart.version>
+        <jsoup.version>1.11.3</jsoup.version>
+        <jsr305.version>3.0.2_1</jsr305.version>
+        <jsr339-api.bundle.version>2.6.0</jsr339-api.bundle.version>
+        <jts.bundle.version>1.14.0_1</jts.bundle.version>
+        <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
+        <jts.version>1.14.0</jts.version>
+        <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
+        <karaf.version>4.2.1</karaf.version>
+        <keyczar.version>0.66</keyczar.version>
+        <la4j.version>0.6.0</la4j.version>
+        <log4j.version>1.2.17</log4j.version>
+        <logback.classic.version>1.2.3</logback.classic.version>
+        <logback.version>1.2.3</logback.version>
+        <lucene.version>7.4.0</lucene.version>
+        <lux.version>1.2</lux.version>
+        <!-- Mina version is not the latest version due to Apache FTPServer 1.0.6 not supporting newer releases properly -->
+        <mina.version>2.0.6</mina.version>
+        <moment.version>2.20.1</moment.version>
+        <mpegts-streamer.version>0.1.0_2</mpegts-streamer.version>
+        <net.minidev.asm.version>1.0.2</net.minidev.asm.version>
+        <netty.version>4.1.24.Final</netty.version>
+        <node.version>v8.9.4</node.version>
+        <objenesis.version>2.5.1</objenesis.version>
+        <ogc-tools-gml-jts.version>1.0.3</ogc-tools-gml-jts.version>
+        <opendj-embedded.app.version>1.3.3</opendj-embedded.app.version>
+        <opengis.bundle.version>${opengis.version}_1</opengis.bundle.version>
+        <opengis.version>19.1</opengis.version>
+        <openlayers.version>4.6.5</openlayers.version>
```

Comment:
❓Front-end? Documentation? Couldn't find where this is being used other than `.adoc` files...

Reply:
Updated